TGC/events/Germany VS Hungary Events.txt

```diff
@@ -0,0 +1,672 @@
+##Treaty of Vienna## 
+
+country_event = {

```

Comment:
I'm wondering if maybe a better approach than just having an event with all the possible ultimatums would be to have a number of decisions for each of the things you can demand (Borderlands, city of Pressburg, autonomy for Banat, autonomy for Siebenburgen, Habsburg restoration) that would each require a certain amount of relations and influence (sphere) with Hungary and which you could ask for individually (and Hungary would be more or less likely to give you stuff depending on what they already gave you and would allow you to more realistically chip away at Hungarian sovereignty.

Reply:
Yes i had the same idea whilst makeing the whole thing but im not quite sure how to implement that.

Reply:
I can see it as 5 decisions for which you need to have Hungary in your sphere:
-One where you get an event where you can choose whether to divide Burgenland or demand it all (the differences being province 3286 and that divide would also move pops, but you wouldn't lose as much influence and no relations). You get cores depending on what you chose and send an event to the Hungarians, who can choose to give it to you or not (with appropriate influence, relations and infamy hits).
-The next one unlocked when you have province 624 to demand Pressburg in a similar way, decision to get a core and send an event. The point of getting cores before an event is that even if Hungary refuses you can still go to war for it later.
-One which would demand that Hungary releases Banat (as a puppet maybe?) and have the primary culture become German like below - if you refuse you can always go to war and then get Banat as your own puppet without Hungary as an accepted culture if you want.
-Similarly for Siebenburgen, maybe a bit of back and forth whether this dominion should include just inner Transylvania or Crisana as well.
-One which would demand that Hungary accept the Habsburg monarchy. Having this would maybe make the above decisions go easier (ai more likely to concede) but would also be difficult to keep in power (maybe a militancy modifier, or more consciousness whenever one of the above demands is accepted). I'm also not 100% sold on this making Hungary a "puppet" but certainly at least for these the flag/modifier would make them more likely to act in Germany's interest.

Don't take this as gospel though, these are just random ideas I'm throwing out. I appreciate the effort going into this and want to see this be the best it can be.

Reply:
Maybe this can be implemented through having two descisions one being "negotiations with hungary?" the other being "an ultimatum to hungary?" the names should be self explainatory. 
During the negotiations i would let the player go trough an event chain where he needs to choose what to demand but not demand too much as that will lead the conference to break down and an event telling you to that sending the ultimatum is the only way to get the land now.

Also maybe the most important context on the hungary puppet part is that the habsburg monarch is likely still king when this happens (as the 1848 revolutionarys wanted to keep his as nominal monarch with an elected government and austria splitting off the non german parts to join germany  will probably result in similar agreements) so imo the restoration of absolute monarchy to hungary in order to make them a client state is not at all unlikely.

Reply:
IMO the point of having more than one decision is that you could wait between making demands in order to build up your influence or relations again, realistically such erosion of territory was always gradual and so I think the player should be given the option to do these at their own convenience.

Reply:
Im actually not a fan of making them influence based or having this thing drawn out over a longer time as i want to add a "unification fervor" modifier for the first 10 years after unification that lets you do this and maybe a reworked baltic german eventchain (if i was allowed to rework that) later. The idea to have it as sepreate descisions is probably better than a blunt eventchain tho.

Reply:
10 years is certainly long enough for what I'm talking about.

Reply:
Maybe, so how would you suggest the influence/relation stuff should work?

Reply:
Something along the lines of you have to have Hungary in your sphere, and if they accept you lose like 50-100 influence and their pops get militancy and consciousness but if they refuse you lose like 100 relations and 200 influence. I think influence would be much better used as a currency than what it is now which is essentially a score, so this is something along those lines, feel free to play around with that.

Reply:
I think i will implement that later but rn the eventchain as it is should suffice when i am done fixing